otelcollector/scripts/setup.sh

```diff
@@ -9,14 +9,20 @@ else
     ARCH=$1
 fi
 
-sudo tdnf install ca-certificates-microsoft -y
-sudo update-ca-trust
+if [ -z $CCP_METRICS_ENABLED ]; then
+  CCP_METRICS_ENABLED="false"
+fi
+echo_var "CCP_METRICS_ENABLED" "$CCP_METRICS_ENABLED"
+
+if [ "${CCP_METRICS_ENABLED}" != "true" ]; then
+    sudo tdnf install ca-certificates-microsoft -y
+    sudo update-ca-trust
+    #Need this for newer scripts
+    chmod 544 $TMPDIR/*.sh
+    chmod 544 $TMPDIR/microsoft/liveness/*.sh
+fi
 
-#Need this for newer scripts
-chmod 544 $TMPDIR/*.sh
-chmod 544 $TMPDIR/microsoft/liveness/*.sh
```

Comment:
why remove  as we aren;t moving liveness to golang yet for main?

Reply:
updated to separate ccpsetup.sh